Tabloid/client/src/modules/tagManager.js

```diff
@@ -0,0 +1,20 @@
+import { getToken } from "./authManager";
+
+const baseUrl = "/api/tag";
+
+export const getAllTags = () => {
+  return getToken().then((token) => {
+    return fetch(baseUrl, {
+          method: "GET",
+          headers: {
+              Authorization: `Bearer ${token}`
```

Comment:
Great job passing in the token to authorize this request! just make sure the API is looking for this authorization on the route for getting all tags.